.github/workflows/build.yml

```diff
@@ -444,7 +444,7 @@ jobs:
           cd ../tests
           npm ci
           npm run build
-          node_modules/.bin/mocha --exit --parallel -j 2 'tracing-tests/**/test-*.js'
+          node_modules/.bin/mocha --exit --parallel -j 2 'tracing-tests/test-*.js'
```

Comment:
Why this change?

Reply:
Because of this error (https://github.com/PureStake/moonbeam/actions/runs/3176272807/jobs/5175556057#step:6:60)  I thought that the path to the tests had changed, but this is not the case.

Reply:
Ok, in fact the error came from the fact that you had modified `*.ts` by `*.js`, I did not see it locally because the script to execute the tracing tests locally had remained in `*.ts.`